src/esp/io/URDFParser.cpp

```diff
@@ -193,11 +207,6 @@ bool Model::loadJsonAttributes(const std::string& filename) {
     return (numConfigSettings > 0);
   }  // if has user_defined tag
 
-  if (!hasRenderAsset) {
-    ESP_WARNING() << "<Model> : Json Config file exists but \"" << subGroupName
-                  << "\" and \"" << attrRenderAsset
-                  << "\" tags not found within file.";
-  }
```

Comment:
Is getting rid of this safe?

Reply:
I don't think that this warning is necessary. Furthermore, we may want to handle these attributes like we handle the others, therefore moving out this code.

I'll leave this warning out.